contracts/deploy/plume/002_core.js

```diff
@@ -0,0 +1,160 @@
+const { deployOnPlume } = require("../../utils/deploy-l2");
+const {
+  deployWithConfirmation,
+  withConfirmation,
+} = require("../../utils/deploy");
+const addresses = require("../../utils/addresses");
+
+module.exports = deployOnPlume(
+  {
+    deployName: "002_core",
+    onlyOnFork: true,
+  },
+  async () => {
+    const { deployerAddr } = await getNamedAccounts();
```

Comment:
you could move much of this into the `deployActions.js` or rather `deployActionsPlume.js`. Though the initial setup won't be repeated in later deploys opposed to Vault and other contract upgrades.

Will leave the decision to you

Reply:
Yeah, I don't think we are gonna redo this. But will extract it when do it